usr/src/uts/armv8/virt/io/virt.c

```diff
@@ -74,8 +74,8 @@ _ischar()
 	return (!(UARTFR & UARTFR_RXFE));
 }
 
-static void _reset(bool poff) __NORETURN;
-static void _reset(bool poff)
+void __NORETURN
```

Comment:
These machine-specific resets are something I want to rework in the not-too-distant future. We have a hard dependency on PSCI (which is not a problem), so there's no real need to hide this away as machine-specific.

Reply:
For the record, that's just an idle comment, not a review comment.